src/utils/GitHubUtils/GitHubUtils.js

```diff
@@ -61,56 +61,58 @@ export const setEmptyCalendarValues = calendar => {
   return copiedCalendar;
 };
 
-export const mergeCalendarsContributions = (
-  actualCalendar,
-  gitHubUserJsonCalendar,
-  timeframe
-) => {
+export const sanitize = calendar =>
```

Comment:
> the structure of the calendars should be simplified somehow(see sanitized(), this basically removes the "noise" from the svg, e.g. attributes, month and weekday texts around the calendar etc)

Note that this introduces noise in the code instead. Is this necessary? I'd prefer simpler code if it can handle "noisy" data.

Reply:
> Note that this introduces noise in the code instead.

I don't think so. Why would we "ditch" the logic if it's way simpler to work with this structure?

Reply:
I'd propose discussing it.

Reply:
> it's way simpler to work with this structure

if that's the case then sure it makes sense